masterlist.yaml

```
@@ -5866,6 +5867,18 @@ plugins:
         subs: [ 'Combat Speechcraft ESP' ]
         condition: 'many("PTCombatSpeechcraft(Easy|Medium|Hard).esp")'
 
+  - name: 'Reworked Race Relations.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/50006/' ]
```

Since the compatibility issue with OCO requires a Bashed Patch or manual patching to be resolved, it would be helpful to notify the user of the issue. It can be hidden if the user has a Bashed Patch active.

```suggestion
    url: [ 'https://www.nexusmods.com/oblivion/mods/50006/' ]
    msg:
      - <<: *patchUnavailable
        subs: [ 'Oblivion Character Overhaul' ]
        condition: 'active("Oblivion_Character_Overhaul.esp") and not active("Bashed Patch.*\.esp")'
```